Code/Framework/AzQtComponents/AzQtComponents/Components/Widgets/AssetFolderThumbnailView.h

```diff
@@ -113,10 +113,10 @@ namespace AzQtComponents
         void setRootIndex(const QModelIndex &index) override;
 
         void SetShowSearchResultsMode(bool searchMode);
+        bool InSearchResultsMode();
```

Comment:
Nit: Add const qualifier here
```suggestion
        bool InSearchResultsMode() const;
```

Reply:
thanks, fixed